client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -643,6 +649,7 @@ const getFormSettings = settings => {
 		'subscriptions',
 		'stb_enabled',
 		'stc_enabled',
+		'gravatar-hovercards',
```

Comment:
If you decide to go with the `<JetpackModuleToggle />` approach I suggested above, you will not need to specifically add the `gravatar-hovercards` to the list of settings fields below, as the module activation state in `<JetpackModuleToggle />` is retrieved directly via the `isJetpackModuleActive` selector.